cmd/main.go

```diff
@@ -60,6 +61,21 @@ func cleanUp(leasePath, tokenPath string) {
 	}
 }
 
+func contactWithEnvVariables(creds vault.Credentials) map[string]string {
```

Comment:
I'd probably change this to something like `appendEnvVars` or `templateWithEnvVars` to make it more obvious what it's doing

Reply:
Hi @Joseph-Irving,
Fixed your suggestion :-)